src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs

```diff
@@ -444,6 +446,7 @@ private static Job CreateAotJob(Job baseJob, CommandLineOptions options, Runtime
                 builder.UseNuGet(ilCompilerVersion, nuGetFeedUrl);
 
             var runtime = runtimeMoniker.GetRuntime();
+            builder.DisplayName(runtime.Name);
             builder.TargetFrameworkMoniker(runtime.MsBuildMoniker);

```

Comment:
This line changed `ILCompiler 6.0.0-*` to `NativeAot6.0`. Revert it


```suggestion
            var runtime = runtimeMoniker.GetRuntime();
            builder.TargetFrameworkMoniker(runtime.MsBuildMoniker);
```